Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutlinerListModel.cpp

```diff
@@ -725,265 +731,78 @@ namespace AzToolsFramework
         return false;
     }
     
-    bool EntityOutlinerListModel::DecodeAssetMimeData(const QMimeData* data, AZStd::optional<ComponentAssetPairs*> componentAssetPairs,
```

Comment:
author note: none of this needs to be here anymore, it is all duplicated from AzAssetBrowser, which now can handle list view drops and thus no longer needs to be duplicated here.